### PR TITLE
[Task]: Remove truncation for the dataset & dataset resource names

### PR DESCRIPTION
**Pull request by Alexandra-Green**
## What this PR accomplishes

- Removes the truncation from the dataset titles on the dataset search page and the resource title on dataset page.
- Adjusts the positioning of the resource list items and explore buttons to accommodate the wrapping of longer resource titles

## Issue(s) addressed

- DATA-1334

## What needs review

- Dataset & dataset resource names are displayed fully.
- Design is responsive (Doesn't cause any issues in different screen sizes)

### CODE REVIEW

**Comment by KatiRG** [comment]
- [ ]     Dataset & dataset resource names are displayed fully.
- [ ]     Design is responsive (Doesn't cause any issues in different screen sizes)

File: ckanext/ontario_theme/templates/internal/snippets/package_list.html [hunk 1]
```diff
@@ -41,8 +41,7 @@
           item_class=item_class,
           hide_resources=hide_resources,
           banner=banner,
-          truncate=truncate,
-          truncate_title=truncate_title
+          truncate=truncate
```
Comment (KatiRG) [inline ckanext/ontario_theme/templates/internal/snippets/package_list.html:44]: do you need this truncate?

File: ckanext/ontario_theme/templates/internal/snippets/package_list.html
Comment (KatiRG) [inline ckanext/ontario_theme/templates/internal/snippets/package_list.html:1]: line 9 typo "trucate"

File: ckanext/ontario_theme/templates/internal/snippets/package_list.html (hunk 1 above)
Comment (Alexandra-Green) [reply ckanext/ontario_theme/templates/internal/snippets/package_list.html:44]: this is for the description truncation

**Review by KatiRG — APPROVED** [review]

**Pull request merged**